package.json

```
@@ -93,7 +93,7 @@
         "playwright": "1.25.0",
         "portastic": "^1.0.1",
         "proxy": "^1.0.2",
-        "puppeteer": "14.4.1",
+        "puppeteer": "~15.1.1",
```

lets keep this pinned, the CI workflow/docker image build might require that
```suggestion
        "puppeteer": "15.1.1",
```